Preferences/OptionalPreference.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  OptionalPreference.swift
+//  Pods
+//
+//  Created by Filip Dolník on 25.05.15.
+//
+//
+
+import Foundation
+
+public class OptionalPreference<T: AnyObject> {
```

Comment:
I think it can't use Preference protocol, because value is optional in this case.